docs/Using-Fleet/MDM-macOS-updates.md

```diff
@@ -61,6 +61,8 @@ Sometimes after the end user clicks "update" on the Nudge window, the end user's
 
 6. In the search bar, type "Software Update." Select **Software Update** and select **Restart Now** to update.
 
+Currently, Nudge does not support [Apple Rapid Security Responses](https://support.apple.com/en-us/HT201224), but this feature is under review and may be included in a future release.  
```

Comment:
```suggestion
Currently, Nudge does not support [Apple Rapid Security Responses](https://support.apple.com/en-us/HT201224), but this feature is under review and may be included in a future release.  
```

Turns out you can use MDM commands to somewhat enforce RSRs. Apple explains how to do this at a high level [here](https://support.apple.com/guide/deployment/rapid-security-responses-dep93ff7ea78/web). 

I think instead of telling the user that the feature might be coming, we should just point them to those Apple docs. This way, the user isn't left hanging. If they want to enforce RSRs they can. They just have to use MDM commands.

@dherder what do you think?

Reply:
@dherder heads up I went ahead and made those changes so we can get this merged in!